### PR TITLE
Convert Paypal transactions to USD

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
@cadecairos r?

File: paypal.js [hunk 1]
```diff
@@ -92,6 +93,11 @@ var detail_q = async.queue(function(task, next) {
     }
 
     task.COUNTRYCODE = transaction.COUNTRYCODE;
+    task.settle_amount = transaction.AMT;
+    if (task.CURRENCYCODE && task.CURRENCYCODE !== 'USD' && transaction.EXCHANGERATE) {
+      task.settle_amount = (transaction.AMT * transaction.EXCHANGERATE * 100 | 0 ) / 100;
```
Comment (cadecairos) [inline paypal.js:98]: Why add the `| 0` in there? could transaction.AMT ever be `undefined` or something?

**Comment by cadecairos** [comment]
R+ with a clarification given on line 98 of paypal.js

File: paypal.js (hunk 1 above)
Comment (jbuck) [inline paypal.js:98]: Stripping the decimal point, so that it only has two decimal points when the calculation is done

**Pull request merged**